api_tests/users/views/user_profile_test_mixin.py

```diff
@@ -193,16 +197,16 @@ def test_create_profile_object(self, app, list_url, user, payload, profile_type)
 
         # test create with start and end dates
         new_name = 'Scundra Town'
-        new_end = '2018-12-01'
-        start_date = payload(institution=new_name, start_date=new_start, end_date=new_end, ongoing=True)
+        new_end = str(datetime.now().year + 1) + '-12-01'
```

Comment:
Can this be done without a dynamic date? That's the kind of thing that causes probabilistically failing tests.

Reply:
It can be. I did this to test my previous functionality of having ongoing=True and a future end date, but since we decided to remove that functionality, this date can be static.